packages/pigeon/example/README.md

```diff
@@ -0,0 +1,96 @@
+# Pigeon iOS / Android Example
+
+## message.dart
+
+```dart
+import 'package:pigeon/pigeon_lib.dart';
+
+class SearchRequest {
+  String query;
+}
+
+class SearchReply {
+  String result;
+}
+
+@HostApi()
+abstract class Api {
+  SearchReply search(SearchRequest request);
+}
+```
+
+## invocation
+
+```sh
+flutter pub run pigeon \
```

Comment:
Does this script check that this needs to happen at the root of a Flutter project? What happens if the Flutter project and the schema and Android and iOS are all sibling directories like in add-to-app?

Reply:
It doesn't need to be run from anywhere in particular except that your pub needs to have access to it.

That scenario shouldn't be a problem.